react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isShown: false };
+    this.cssClass = 'side-menu';
```

Comment:
In general, we want to avoid storing extra properties on `this`. A timer id is a special case, but usually extra properties on `this` are not necessary or recommended.

Reply:
Placed the css variable in render